src/main/java/com/commercetools/paymenttoorderprocessor/jobs/actions/OrderCreator.java

```diff
@@ -30,41 +41,67 @@
  * @author mht@dotsource.de
  *
  */
-public class MessageWriter implements ItemWriter<Cart> {
-    public static final Logger LOG = LoggerFactory.getLogger(MessageWriter.class);
+public class OrderCreator implements ItemWriter<CartAndMessage> {
+    private static final Logger LOG = LoggerFactory.getLogger(OrderCreator.class);
 
     private static final String ENCRYPTIONALGORITHM = "Blowfish";
+    private static final int DEFAULTTIMEOUT = 40000;
 
     @Value("${createorder.encryptionkey}")
```

Comment:
as mentioned before: isn't it better to keep it consistent with other projects (FI, `project-coeur-java:CREATEORDER_ENCRYPTION_SECRET`)